manual_tests/manual_test_intervals.py

```diff
@@ -202,8 +202,8 @@
 
 # reference="left", right_open,
 ref_references = [1, 2, 3, 4, 5]
-ref_intervals = [[1.2], [1.5, 2.4], [2.5, 2.6, 3.1], [3.5, 3.6, 4.0], [5.0]]
-ref_boundaries = [[0.5, 1.5], [1.5, 2.5], [2.5, 3.5], [3.5, 4.5], [4.5, 5.5]]
+ref_intervals = [[1.2, 1.5], [2.4, 2.5, 2.6], [3.1, 3.5, 3.6], [4.0], [5.0]]
```

Comment:
Is this datastructure correct? ref_intervals = [[1.2, 1.5], [2.4, 2.5, 2.6], [3.1, 3.5, 3.6], [4.0], [5.0]]
What does it describe? If these are the reference values for intervals why is not one value per interval? E.g. the middle of the interval?

Probably I should know that as we probably together discussed it (?) but at the moment it does not make any sense to me.

Reply:
The datapoint are sorted into ref_intervals (the naming of the variables are confusing). The references of the intervals are indicated in re_references.

Reply:
@kawindme should we delete this manual test?